specification/keyvault/Security.KeyVault.Secrets/client.tsp

```diff
@@ -8,9 +8,6 @@ using Azure.ClientGenerator.Core;
 
 using KeyVault;
 
-// generate DeletionRecoveryLevel in JS
-@@usage(DeletionRecoveryLevel, Usage.output, "javascript");
-
 @@clientName(SecretProperties, "SecretP", "go");
 @@clientName(DeletedSecretBundle, "DeletedSecret", "go");
 @@clientName(DeletedSecretItem, "DeletedSecretProperties", "go");
```

Comment:
This is the same for every language, or, @chlowell, is the thinking that everyone else will only generate underlying APIs and still maintain the hand-coded wrappers?

Reply:
Yes, that's the plan: we'll keep all the existing wrappers because emitters don't output them and we need to maintain compatibility with our current stable APIs